docs/podman-create.1.md

```diff
@@ -475,6 +475,10 @@ its root filesystem mounted as read only prohibiting any writes.
 
 Automatically remove the container when it exits. The default is *false*.
 
+Note that the container will not be removed in case it could not be created or
+started successfully. This allows the user to inspect the container after
+failure. The `--rm` flag is incompatible with with the `-d` flag.
+
```

Comment:
Suggest:  "Note that the container will not be removed when it could not be created or started successfully."

"with with" to "with"

Reply:
I must be blind today ... thanks a lot!

Reply:
Sadly I missed it also.